communities/UNAM-Engineering/community-assessment.md

```diff
@@ -0,0 +1,162 @@
+# UNAM Engineering 
+[UNAM Engineering](https://github.com/unamfi) is a small Github organization started on 2016 with the objective to provide students a space
```

Comment:
Thanks! That actually is a great idea! It could help others create similar GH Orgs for their university